src/renderer/components/team/dialogs/TaskAttachments.tsx

```
@@ -73,48 +82,74 @@ export const TaskAttachments = ({
+    () =>
+      imageAttachments.map((a) => ({
+        src: thumbCache.get(a.id) ?? PLACEHOLDER_SRC,
+        alt: a.filename,
+      })),
+    [imageAttachments, thumbCache]
+  );
+
+  const handlePreview = useCallback(
+    (att: TaskAttachmentMeta) => {
+      if (!isImageMimeType(att.mimeType)) {
+        void handleDownload(att);
```

ESLint

[error] Remove this use of the "void" operator.

(sonarjs/void-use)